src/universal/types/schema.flow.js

```diff
@@ -0,0 +1,1705 @@
+  /** The body of the agenda item */
+  content: string;
+  /** The timestamp the agenda item was created */
+  createdAt: ?any;
+  /** true until the agenda item has been marked isComplete and the meeting has ended */
+  isActive: ?boolean;
+  /** true if the agenda item has been addressed in a meeting (will have a strikethrough or similar) */
+  isComplete: ?boolean;
+  /** The sort order of the agenda item in the list */
+  sortOrder: ?number;
+  /** *The team for this agenda item */
+  teamId: string;
+  /** The teamMemberId that created this agenda item */
+  teamMemberId: string;
+  /** The timestamp the agenda item was updated */
+  updatedAt: ?any;
+  /** The team member that created the agenda item */
+  teamMember: ?TeamMember;
+}
+
+/**
+  A member of a team
+*/
+export type TeamMember = {
+  /** An ID for the teamMember. userId::teamId */
+  id: ?string;
+  /** true if the user is a part of the team, false if they no longer are */
+  isNotRemoved: ?boolean;
+  /** Is user a team lead? */
+  isLead: ?boolean;
+  /** Is user a team facilitator? */
+  isFacilitator: ?boolean;
+  /** hide the agenda list on the dashboard */
+  hideAgenda: ?boolean;
+  /** The user email */
+  email: ?any;
+  /** url of user’s profile picture */
+  picture: ?any;
+  /** The name, as confirmed by the user */
+  preferredName: ?string;
+  /** The place in line for checkIn, regenerated every meeting */
+  checkInOrder: ?number;
+  /** true if the user is connected */
+  isConnected: ?boolean;
+  /** true if present, false if absent, null before check-in */
+  isCheckedIn: ?boolean;
+  /** true if this team member belongs to the user that queried it */
+  isSelf: ?boolean;
+  /** foreign key to Team table */
+  teamId: ?string;
+  /** foreign key to User table */
+  userId: ?string;
+  /** The team this team member belongs to */
+  team: ?Team;
+  /** The user for the team member */
+  user: ?User;
+  /** Projects owned by the team member */
+  projects: ?ProjectConnection;
+}
+
+/**
+  A connection to a list of items.
+*/
+export type ProjectConnection = {
+  /** Page info with cursors coerced to ISO8601 dates */
+  pageInfo: ?PageInfoDateCursor;
+  /** A list of edges. */
+  edges: ?Array<ProjectEdge>;
+}
+
+/**
+  Information about pagination in a connection.
+*/
+export type PageInfoDateCursor = {
+  /** When paginating forwards, are there more items? */
+  hasNextPage: boolean;
+  /** When paginating backwards, are there more items? */
+  hasPreviousPage: boolean;
+  /** When paginating backwards, the cursor to continue. */
+  startCursor: ?any;
+  /** When paginating forwards, the cursor to continue. */
+  endCursor: ?any;
+}
+
+/**
+  An edge in a connection.
+*/
+export type ProjectEdge = {
+  /** The item at the end of the edge */
+  node: ?Project;
+  cursor: ?any;
+}
+
+/**
+  A long-term project shared across the team, assigned to a single user 
+*/
+export type Project = {
+  /** A shortid for the project teamId::shortid */
+  id: ?string;
```

Comment:
So my problem right now with relying on these autogenerated types is that they're not terribly precise.
- Every single field, including `id`, is a maybe type, which is not good. Flow will yell at us every time we try to do something with `project.id` unless we account for its possibly empty value.
- `updatedAt: ?any` is no better than forgoing a type system entirely, since `?any` is the broadest possible set of values.

Reply:
we could, any maybe should, make `id` nonnull in GraphQL.that'd fix the first issue.
`updatedAt` is an edge case because I use my own custom scalar to handle dates. there's a PR out there to fix this